test/adapter/json/collection_test.rb

```diff
@@ -85,6 +85,20 @@ def test_include_option
 
             assert_equal(expected, actual)
           end
+
+          def test_fields_with_no_associations_include_option
+            serializer = ArraySerializer.new([@first_post, @second_post])
+            adapter = ActiveModel::Serializer::Adapter::Json.new(serializer)
+            actual = adapter.serializable_hash(fields: [:id])
```

Comment:
The option should not be passed to `serializable_hash` but to the adapter's constructor.